build.rs

```diff
@@ -1,26 +1,213 @@
+        // calibnet
+        (
+            Cid::try_from("bafy2bzacedbedgynklc4dgpyxippkxmba2mgtw7ecntoneclsvvl4klqwuyyy").unwrap(),
+            "https://forest-continuous-integration.fra1.cdn.digitaloceanspaces.com/builtin-actors/calibnet/Shark.car",
+        ),
+        (
+            Cid::try_from("bafy2bzaced25ta3j6ygs34roprilbtb3f6mxifyfnm7z7ndquaruxzdq3y7lo").unwrap(),
+            "https://forest-continuous-integration.fra1.cdn.digitaloceanspaces.com/builtin-actors/calibnet/Hygge.car",
+        ),
+        (
+            Cid::try_from("bafy2bzacedhuowetjy2h4cxnijz2l64h4mzpk5m256oywp4evarpono3cjhco").unwrap(),
+            "https://forest-continuous-integration.fra1.cdn.digitaloceanspaces.com/builtin-actors/calibnet/Lightning.car",
+        ),
+        // devnet
+        (
+            Cid::try_from("bafy2bzacebzz376j5kizfck56366kdz5aut6ktqrvqbi3efa2d4l2o2m653ts").unwrap(),
+            "https://forest-continuous-integration.fra1.cdn.digitaloceanspaces.com/builtin-actors/devnet/Hygge.car",
+        ),
+        (
+            Cid::try_from("bafy2bzaceay35go4xbjb45km6o46e5bib3bi46panhovcbedrynzwmm3drr4i").unwrap(),
+            "https://forest-continuous-integration.fra1.cdn.digitaloceanspaces.com/builtin-actors/devnet/Lightning.car",
+        ),
+        // mainnet
+        (
+            Cid::try_from("bafy2bzaceb6j6666h36xnhksu3ww4kxb6e25niayfgkdnifaqi6m6ooc66i6i").unwrap(),
+            "https://forest-continuous-integration.fra1.cdn.digitaloceanspaces.com/builtin-actors/mainnet/Shark.car",
+        ),
+        (
+            Cid::try_from("bafy2bzacecsuyf7mmvrhkx2evng5gnz5canlnz2fdlzu2lvcgptiq2pzuovos").unwrap(),
+            "https://forest-continuous-integration.fra1.cdn.digitaloceanspaces.com/builtin-actors/mainnet/Hygge.car",
+        ),
+        (
+            Cid::try_from("bafy2bzacecnhaiwcrpyjvzl4uv4q3jzoif26okl3m66q3cijp3dfwlcxwztwo").unwrap(),
+            "https://forest-continuous-integration.fra1.cdn.digitaloceanspaces.com/builtin-actors/mainnet/Lightning.car",
+        ),
+    ] {
+        tasks.spawn(async move {
+            download_bundle_if_needed(root, url).await.with_context(|| format!("Failed to get {root}.car from {url}"))
+        });
+    }
+
+    let mut car_roots = vec![];
+    let mut car_readers = vec![];
+    while let Some(path) = tasks.join_next().await {
+        let car_reader = fvm_ipld_car::CarReader::new(futures::io::BufReader::new(
+            async_fs::File::open(path??).await?,
+        ))
+        .await?;
+        car_roots.extend_from_slice(car_reader.header.roots.as_slice());
+        car_readers.push(car_reader);
+    }
+
+    let car_writer = CarHeader::from(
+        car_readers
+            .iter()
+            .flat_map(|it| it.header.roots.iter())
+            .unique()
+            .cloned()
+            .collect::<Vec<_>>(),
+    );
+
+    let mut zstd_encoder = ZstdEncoder::with_quality(
+        async_fs::File::create(Path::new(&env::var("OUT_DIR")?).join("actor_bundles.car.zst"))
```

Comment:
This is also defined in `src/daemon/bundle.rs`. Let's have a single source of truth.

Reply:
Not sure how to share the code, `include_bytes!(concat!(env!("OUT_DIR"), "/actor_bundles.car.zst"));` requires str literals and is evaluated at compile time

Reply:
Below code does not compile, only literals (like `"foo"`, `-42` and `3.14`) can be passed to `concat!()`
```rust
const BUNDLE_NAME: &str = "/actor_bundles.car.zst";
pub const ACTOR_BUNDLES_CAR_ZST: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), BUNDLE_NAME));
```